lib/apiservers/engine/backends/container.go

```diff
@@ -165,14 +164,13 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 		config.Config.Entrypoint = layer.Config.Entrypoint
 	}
 	config.Config.Env = append(config.Config.Env, layer.Config.Env...)
-
 	log.Printf("config.Config' = %+v", config.Config)
 
 	// Call the Exec port layer to create the container
-	host, err := guest.UUID()
+	host, err := os.Hostname()
```

Comment:
I think this is wrong as it reverts @fdawg4l's change (#79df40b586ba5191844068c624814dfde0a2d016)